adobe/react-spectrum
# Allow passing data attributes to FieldError

Closes #6888 

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Run Storybook locally
2. Add a `data-` attribute to react-aria `TextField` story's `FieldError`
3. In the browser, open React Aria Components > Text Field Submit Example
4. Click Submit button with an empty input 
5. Inspect error message to ensure that it contains the `data-` attribute 

<img width="2560" alt="Screenshot 2024-08-16 at 10 22 55 AM" src="https://github.com/user-attachments/assets/2181986a-8d2a-4cba-8f4a-855521a74d82">


## 🧢 Your Project:
